packages/console/src/pages/EnterpriseSsoDetails/index.tsx

```diff
@@ -66,7 +71,7 @@ function EnterpriseSsoConnectorDetails<T extends SsoProviderName>() {
               .identityProvider
         ));
 
-  const isLoading = !ssoConnector && !requestError;
+  const isLoading = isSsoConnectorLoading || isUserAssetServiceLoading;
```

Comment:
Might need to make sure the fullscreen loading won't show up again when reloading the data?

Reply:
Since we have loaded data.